packages/flutter/lib/src/rendering/wrap.dart

```diff
@@ -392,11 +391,10 @@ class RenderWrap extends RenderBox with ContainerRenderObjectMixin<RenderBox, Wr
     while (child != null) {
       final double childWidth = child.getMaxIntrinsicWidth(double.infinity);
       final double childHeight = child.getMaxIntrinsicHeight(childWidth);
-      if (runWidth + childWidth > width) {
-        height += runHeight;
-        if (runCount > 0)
-          height += runSpacing;
-        runCount += 1;
+      final double spaceWidth = childCount > 0 ? spacing : 0;
```

Comment:
you can remove this space width calculation. If `childCount == 0`, it will fail the condition check at line 396 anyway.